tests/utils/test_audio_utils.py

```diff
@@ -762,56 +1606,80 @@ def test_amplitude_to_db(self):
         with pytest.raises(ValueError):
             amplitude_to_db(spectrogram, db_range=-80)
 
-    @require_librosa
-    def test_chroma_equivalence(self):
```

Comment:
I think you removed this test, probably a mistake right ? In that case, let's put it back and reimport librosa !